### ISSUE TITLE
Sitting in a non-uniformed scaled anchor scales your local mode dashboard

### ISSUE THREAD

**Issue by AshtonSparx** [Neos-Metaverse/NeosPublic#2799]
### Describe the bug?

While sitting in an anchor that's not scaled properly. Your dashboard's visuals get scaled with it.

### Relevant issues

None I could find

### To Reproduce

- Scale an anchor on 1 or 2 axis
- Get into the anchor
- observe the visuals being shrunk/enlarged

### Expected behavior

For anchors to not effect your local dashboard

### Screenshots

Scaling down (1/2 axis being shrunk)
![image](https://user-images.githubusercontent.com/53165730/129464178-0270bc9e-2ba5-41ec-aac0-ff810e25d57b.png)

Scaling it up (1/2 axis)
![image](https://user-images.githubusercontent.com/53165730/129464204-d3e8b459-fc77-49b7-af20-60d69cb83fde.png)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.8.14.443

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Additional context

Unsure if it worked on previous builds since normally.. everyone scales anchors properly. Only noticed it in someone's world.

### Reporters

AshtonSparx#5050

**Comment by Frooxius** [comment]
We might be able to fix this particular case, but generally you really **REALLY** shouldn't use non-uniform scaling, especially on a player, it's going to break a bunch of things.

The best solution here might be to just automatically pop the player out if they're non-uniformly scaled and prevent them from entering in such case (with small tolerance to prevent cases like [1, 0.99999998, 0.9999998]).

**Comment by shadowpanther** [comment]
Also happens while being parented to a non-uniform slot in general, as reported by Enverex:
![Image](https://media.discordapp.net/attachments/731903736028332083/872918744916164639/unknown.png?width=2082&height=1139)
https://discord.com/channels/402159838827905024/731903736028332083/872918749349543976

**Comment by AshtonSparx** [comment]
> 
> 
> We might be able to fix this particular case, but generally you really **REALLY** shouldn't use non-uniform scaling, especially on a player, it's going to break a bunch of things.
> 
> The best solution here might be to just automatically pop the player out if they're non-uniformly scaled and prevent them from entering in such case (with small tolerance to prevent cases like [1, 0.99999998, 0.9999998]).

Would there be a way to prevent non-uniformed scaled anchors in the first place? Unsure of any methods currently other than it auto offsetting it's own scale nut that could cause issues(?)

**Comment by Frooxius** [comment]
@shadowpanther Yeah, this is just result of non-uniform scaling of the hierarchy, not anything specific to the anchors. 

@AshtonSparx Well preventing it from being used when non-uniformly scaled is one way of preventing them. Other than that... don't use non-uniform scale. We could do something that ensures uniform scale in the hierarchy in this case, but that might just end up breaking your stuff, it's more responsibility of the creator to ensure their hierarchy is good.
